tools/sidekick/config.js

```diff
@@ -25,17 +25,24 @@ function hasSchema(host) {
     libraries: [
```

Comment:
config.js is the old way to configure the sidekick.  No need to update this file as it will be removed soon.

Reply:
Reverted ✅